module/VuFind/src/VuFind/Db/Row/UserFactory.php

```diff
@@ -82,7 +82,6 @@ public function __invoke(
         $ilsAuthenticator = $container->get(\VuFind\Auth\ILSAuthenticator::class);
         $favoritesService = $container->get(FavoritesService::class);
         $prototype = parent::__invoke($container, $rowClass, [$ilsAuthenticator, $capabilities, $favoritesService]);
-        $prototype->setConfig($config);
```

Comment:
Config is no longer used, so why bother setting it?

Reply:
`$prototype` isn't used either, so could just return the `parent::__invoke()` also.